libseat-sys/build.rs

```diff
@@ -25,4 +32,5 @@ fn main() {
 #[cfg(not(feature = "use_bindgen"))]
 fn main() {
     println!("cargo:rustc-link-lib=seat");
```

Comment:
Is this still needed? As far as I can tell, pkg_config will link the lib in `probe_library() -> Config::probe() -> Library::parse_libs_cflags()`.
Same goes for bindgen build

Feel free to correct me if I missed the purpose somehow.

Reply:
Right, it seems the `pkg_config` adds this automatically.

Updated.